build-bin/docker-compose-kafka.test.yml

```diff
@@ -0,0 +1,65 @@
+# permit depends_on/condition: service_healthy
+version: "2.4"
+
+# this duplicates a lot of service definitions because services with depends_on cannot be extended
+services:
+  sut:
+    container_name: sut
+    image: ghcr.io/openzipkin/alpine:3.12.3
+    entrypoint: /bin/sh
+    # Keep the container running until HEALTHCHECK passes
+    command: "-c \"sleep 5m\""
+    healthcheck:
+      # Return 0 when we can read back our trace ID
+      test: wget -qO- --spider http://zipkin:9411/api/v2/trace/cafebabecafebabe
```

Comment:
it occurs to me this doesn't actually test that backend sent a span (I checked and it does), but yeah.. we might want to change the test to include `| grep ...` where ... is something that verifies backend is the "local" servicename. we could install lazy jq possibly, but it is probably fine to just use grep. That, or we can count the spans.. ex |grep spanId|uniq or something

Reply:
cc @jcchavezs noting this isn't important to address here (possibly ever)